### PR TITLE
I/6110: Change name LabeledView -> LabeledFieldView

### PR DESCRIPTION
**Pull request by panr**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Other: Changed the name of the `LabeledView` component to the `LabeledFieldView`. See ckeditor/ckeditor5#6110.

Related PR: https://github.com/ckeditor/ckeditor5-theme-lark/pull/274

--
### BREAKING CHANGES:

- From now the instance of a labeled component's view is available through `LabeledFieldView#field`. It replaced `LabeledView#view`.

### CODE REVIEW

**Comment by panr** [comment]
FYI: CI breaks due to the fact that I use `console.warn`.

**Comment by panr** [comment]
I'm struggling with CI. Tried almost everything... What I'm missing? 🤔

File: src/labeledfieldview/labeledfieldview.js [hunk 1]
```diff
@@ -175,7 +175,7 @@ export default class LabeledView extends View {
 			},
 			children: [
 				this.labelView,
-				this.view,
+				this.field,
```
Comment (jodator) [inline src/labeledfieldview/labeledfieldview.js]: Should be `this.fieldView` as it hurts when you read those 3 in a row ;) We should keep property naming consinstent.

Comment (panr) [reply src/labeledfieldview/labeledfieldview.js]: Hmm... I had a talk with @Reinmar and @oleq about the naming and the suggestion was to use `this.field`, but right now I'm not sure... `this.fieldView` looks/feels better IMHO. (sooooo many changes all over again 😵)

Comment (jodator) [reply src/labeledfieldview/labeledfieldview.js]: OH I didn't know about that :( But looking at other props `fieldView` looks better :/). You can always rename others to match  :P

Comment (Reinmar) [reply src/labeledfieldview/labeledfieldview.js]: fieldView :+1:

Comment (panr) [reply src/labeledfieldview/labeledfieldview.js]: OK, I'm on it.

**Comment by panr** [comment]
@jodator all done ✅🤞

**Pull request merged**